src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+  const properties = sourceString.split(';').map((prop) => prop.split(':'));
```

Comment:
[CODE KNOWLEDGE] - Don't need to make a copy before each operation. Moreover, methods like `map` don't mutate the original array. You can chain `map` directly after `split` without using an intermediate variable.